src/App.vue

```diff
@@ -1,26 +1,44 @@
 <template>
   <v-app v-mousetrap="mousetrapBindings" id="inspire">
     <v-app-bar class="elevation-1" app clipped-right>
-      <v-app-bar-nav-icon @click.stop="drawer = !drawer" />
       <v-toolbar-title @click="goHome" class="logo"
         >NimbusImage</v-toolbar-title
       >
       <bread-crumbs />
       <v-spacer />
-      <v-btn @click.stop="toggleRightPanel('snapshotPanel')" id="snapshotButton"
-        >Snapshot</v-btn
-      >
-      <user-menu />
-      <v-btn @click.stop="toggleRightPanel('annotationPanel')"
-        >Browse Annotations</v-btn
+      <v-btn
+        color="primary"
+        class="ml-4"
+        :to="{ name: 'newdataset' }"
```

Comment:
newdataset -> newDataset?

Reply:
This is the name of the route I took from src/views/dataset/index.ts

```
path: "/new",
name: "newdataset",
component: NewDataset,
```

```
path: "/import",
name: "importdataset",
component: ImportDataset,
```

Should I refactor "newdataset" and "importdataset" ?

Reply:
Ohhhh I see.
No you don't need to change anything!